src/components/numeral-date/numeral-date.spec.tsx

```diff
@@ -691,25 +686,4 @@ describe("NumeralDate", () => {
       });
     });
   });
-
-  it("renders with the expected border radius styling on first and last inputs", () => {
```

Comment:
**suggestion(non-blocking):** We'll probably still need a test asserting the border-radius which has been changed to `border-radius: var(--borderRadius050);`